portality/templates/doaj/contact.html

```diff
@@ -45,8 +45,16 @@
     {{ render_field_horizontal(form.message, style="width: 100%; max-width: 400px; min-height: 200px", placeholder="Enter your message here") }}
 
     <div class="form-group">
-        <div class="col-md-9 col-md-offset-3">
-            <button class="btn btn-success" id="send">Send</button>
+        <div class="submit-with-recaptcha col-md-9 col-md-offset-2">
+            <form method="POST">
+              <div id="html_element"></div>
+            </form>
+            <script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit"
```

Comment:
If we modify the contact.js file in the way I suggested, then we also need to just change `onload=doaj.contact.onLoadCallback`

Reply:
We could. But `onload=doaj.contact.onLoadCallback` is not accepted. The workaround would be to  do something like:
```
var script = document.createElement('script');

script.id = 'container'
script.src = '//www.google.com/recaptcha/api.js?render=explicit';
script.async = true;
script.defer = true;

script.onload = () => { ... };

document.body.appendChild(script);
```
in the contact.js file in `jQuery(document).ready(function($) `. I am not really sure if that will improve that code. And, tbh, I've struggled a bit to make it work. Could we maybe pair up sometime and have a look at it together?